app/models/charge.rb

```diff
@@ -59,4 +61,12 @@ def validate_charge_model(validator)
       .flatten
       .each { |code| errors.add(:properties, code) }
   end
+
+  def validate_group_properties
+    # Group properties should be set for all the selectable groups of a BM
+    bm_group_ids = billable_metric.selectable_groups.pluck(:id).sort
```

Comment:
It would be more efficient to sort it on DB level (especially if we have right indexes set) but probably it's fine to sort it in this example in memory also (if there are small umber of records)

Reply:
Yeah, I think it's acceptable for now (as we can have only two dimensions) but I agree with you